cvat-ui/src/components/annotation-page/standard-workspace/context-image/context-image.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { ArrowsAltOutlined, ShrinkOutlined } from '@ant-design/icons';
+import Spin from 'antd/lib/spin';
+import Image from 'antd/lib/image';
+
+import { CombinedState } from 'reducers/interfaces';
+import { hideShowContextImage, getContextImage } from 'actions/annotation-actions';
+import CVATTooltip from 'components/common/cvat-tooltip';
+
+export default function ContextImage(): JSX.Element | null {
+    const dispatch = useDispatch();
+    const { number: frame, hasRelatedContext } = useSelector((state: CombinedState) => state.annotation.player.frame);
+    const { data: contextImageData, hidden: contextImageHidden, fetching: contextImageFetching } = useSelector(
+        (state: CombinedState) => state.annotation.player.contextImage,
+    );
+    const [requested, setRequested] = useState(false);
+
+    useEffect(() => {
+        if (requested) {
+            setRequested(false);
+        }
+    }, [frame]);
+
+    useEffect(() => {
+        if (hasRelatedContext && !contextImageHidden && !requested) {
+            dispatch(getContextImage());
+            setRequested(true);
+        }
+    }, [contextImageHidden, requested, hasRelatedContext]);
+
+    if (!hasRelatedContext) {
+        return null;
+    }
+
+    return (
+        <div className='cvat-context-image-wrapper' {...(contextImageHidden ? { style: { width: '40px' } } : {})}>
+            <div className='cvat-context-image-wrapper-header' />
```

Comment:
In my opinion it's can be better to cut this element by width because it's cover part of the image without special reason.

![изображение](https://user-images.githubusercontent.com/8168727/117273707-0beddd00-ae65-11eb-848e-e768efa8773c.png)

Reply:
It was done according to antd Image element. 
It also has a transparent black line at the top of the image with buttons at right.


![Screenshot from 2021-05-06 12-29-02](https://user-images.githubusercontent.com/40690378/117275443-b74b6180-ae66-11eb-8219-15ed609283e9.png)